Unfortunately there are still two pylint issues. 

```bash
./homeassistant/components/camera/generic.py:21:1: E302 expected 2 blank lines, found 1
./homeassistant/components/camera/mjpeg.py:24:1: E302 expected 2 blank lines, found 1
```

--- a/homeassistant/components/camera/generic.py
+++ b/homeassistant/components/camera/generic.py
@@ -17,6 +17,7 @@ _LOGGER = logging.getLogger(__name__)
 BASIC_AUTHENTICATION = 'basic'
 DIGEST_AUTHENTICATION = 'digest'
 
+
 # pylint: disable=unused-argument
 def setup_platform(hass, config, add_devices_callback, discovery_info=None):
     """Setup a generic IP Camera."""

--- a/homeassistant/components/camera/mjpeg.py
+++ b/homeassistant/components/camera/mjpeg.py
@@ -20,6 +20,7 @@ _LOGGER = logging.getLogger(__name__)
 BASIC_AUTHENTICATION = 'basic'
 DIGEST_AUTHENTICATION = 'digest'
 
+
 # pylint: disable=unused-argument
 def setup_platform(hass, config, add_devices_callback, discovery_info=None):
     """Setup a MJPEG IP Camera."""

